templates/podcast-episode.html.twig

```diff
@@ -7,7 +7,12 @@
     {% endif %}
     <h3>Posted {{ header.date|date('Y-m-d')|nicetime(false) }}</h3>
     {{ page.content }}
-    {{ page.media[ (page.header.podcast.audio|first).name ].html() }}
+    {% if page.header.podcast.audio %}
+        {% set audio_file = page.header.podcast.audio|first.name  %}
+    {% elseif page.header.podcast.audio %}
```

Comment:
This condition is the same as L10 above?

Reply:
if page.header.podcast.audio
vs
elseif page.header.podcast.audio

Reply:
Ooh... yea. I see what  you are saying. I think its supposed to say `page.header.podcast.audio_url`